docs/HowTo/Find-and-Connect/Specifying-NAT.md

```diff
@@ -1,64 +1,72 @@
-When `DOCKER` is specified, the host IP address is advertised not the container IP address. 
+* [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host) and
+  [`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port) define the advertised host and port
+  for the P2P service.
+* [`--rpc-http-host`](../../Reference/CLI/CLI-Syntax.md#rpc-http-host) and
+  [`rpc-http-port`](../../Reference/CLI/CLI-Syntax.md#rpc-http-port) define the advertised host and
+  port for the JSON-RPC service.
 
-The host IP is the advertised host specified in the [`docker run` command](https://docs.docker.com/engine/reference/commandline/run/#add-entries-to-container-hosts-file---add-host). 
-If not specified in the `docker run` command, the advertised host defaults to [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host) 
-and [`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port). 
+## Docker
 
-## Auto 
+Specify `DOCKER` to explicitly specify Hyperledger Besu is running inside a Docker container. If
+you specify `DOCKER`, you advertise the host IP address not the container IP address.
 
-`AUTO` is the default NAT method. `AUTO` detects if Besu is running inside Docker container. If 
-inside a Docker container, acts as if [`DOCKER`](#docker) is specified. Otherwise, acts as if 
-[`NONE`](#none) is specified. 
+The host IP is the advertised host specified in the
+[`docker run` command](https://docs.docker.com/engine/reference/commandline/run/#add-entries-to-container-hosts-file---add-host).
+If not specified in the `docker run` command, the advertised host defaults to the values for
+[`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host) and
+[`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port).
 
-## None 
+## Auto
 
-Specify `NONE` to explicitly specify Besu does not use NAT. If the NAT method is `NONE` and [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host), 
-[`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port), [`--rpc-http-host`](../../Reference/CLI/CLI-Syntax.md#rpc-http-host), 
-and [`rpc-http-port`](../../Reference/CLI/CLI-Syntax.md#rpc-http-port) are specified, they are 
-ignored in the advertised addresses and the default values are advertised. 
+`AUTO` is the default NAT method. `AUTO` detects if Besu is running inside Docker container. If
+inside a Docker container, `AUTO` acts as if you specified [`DOCKER`](#docker). If not inside a
+Docker container, `AUTO` acts as if you specified [`NONE`](#none).
 
+## None
 
+Specify `NONE` to explicitly specify Besu does not use NAT. If the NAT method is `NONE`, Besu
+ignores any values specified in the [`--p2p-host`](../../Reference/CLI/CLI-Syntax.md#p2p-host),
+[`--p2p-port`](../../Reference/CLI/CLI-Syntax.md#p2p-port),
+[`--rpc-http-host`](../../Reference/CLI/CLI-Syntax.md#rpc-http-host), and
+[`--rpc-http-port`](../../Reference/CLI/CLI-Syntax.md#rpc-http-port) options, with the default
+values advertised instead.
```

Comment:
"and the default
values are advertised instead." ?

Reply:
Broke it up into two sentences. 
"... options. Besu advertises the default values instead."